.changeset/healthy-pigs-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': major
```

Comment:
Since it is a major change, do we have to include it in the migration guide? (I noticed you wrote "n/a" under "Docs")

Reply:
i think it's not that important because the class will do nothing if it's still present in the markup. but i'll make a brief note in the migration guide.